plugin/pkg/auth/authorizer/rbac/bootstrappolicy/namespace_policy.go

```diff
@@ -93,6 +93,15 @@ func init() {
 			eventsRule(),
 		},
 	})
+	addNamespaceRole(metav1.NamespaceSystem, rbac.Role{
+		// role for the leader locking on supplied configmap
+		ObjectMeta: metav1.ObjectMeta{Name: "system::leader-locking"},
+		Rules: []rbac.PolicyRule{
```

Comment:
The "update" permission must be scoped to a particular configmap name.  There is at least one configmap in kube-system which is safe to read, but which is unsafe to mutate.